.github/workflows/release.yml

```diff
@@ -46,3 +46,42 @@ jobs:
           asset_path: ${{ steps.get_version.outputs.WAR_FILE}}
           asset_name: ${{ steps.get_version.outputs.WAR_FILE}}
           asset_content_type: application/rpm
+  publish:
+    name: API Container Image
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+    # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+      - name: get RADAR version
+        id: get_version
+        run: .github/workflows/get_version.sh
+      - name: Configure AWS credentials
+            if: ${{ always() }}
+            uses: aws-actions/configure-aws-credentials@v1
+            with:
+              aws-access-key-id: ${{ secrets.CWBICI_DEVELOP_AWS_ACCESS_KEY_ID }}
+              aws-secret-access-key: ${{ secrets.CWBICI_DEVELOP_AWS_SECRET_ACCESS_KEY }}
+              aws-region: ${{ secrets.CWBICI_DEVELOP_AWS_REGION }}
+       name: Login to Amazon ECR
+        if: ${{ success() }}
+        id: login-ecr
+        uses: aws-actions/amazon-ecr-login@v1
+
+      - name: Build, tag, and push image to Amazon ECR (midas-api)
+        if: ${{ success() }}
+        env:
+          ECR_REGISTRY: ${{ steps.login-ecr.outputs.registry }}
+          ECR_REPOSITORY: cwms-radar-api
+          IMAGE_TAG: ${{steps.get_version.outputs.VERSION}}
+        run: |
+          ./gradlew prepareDocker
+          cd cwms_radar_standlone/build/docker
+          docker build -t $ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG -t $ECR_REGISTRY/$ECR_REPOSITORY:latest .
```

Comment:
Hi @MikeNeilson  - I think this would probably work, but I typically use a slightly different approach.  I believe as-written, this builds the `.jar` in the actions worker environment (outside of the selected docker container) then copies the resulting `.jar` into the docker container.  Recommend running the build inside a container and using [multi-stage builds](https://docs.docker.com/develop/develop-images/multistage-build/) to only copy the `.jar` of interest into the final image and keep image size small.

Reply:
The base image used is the openjdk11 and and uber jar that's 33MB The docker build is only going to add the single 33 MB jar. So while I don't disagree with doing the multi-stage in general, I don't think it's going to do much to help shrink anything. Though if proven wrong we can certainly change it later.

Thanks for the check though. Once someone approves it I'll merge it in and we'll see what happens.

Reply:
Make that adoptopenjdk:11-jre-hotspot, if I'm going to identify a docker image I should identify the one I'm actually using.

Reply:
>  I don't think it's going to do much to help shrink anything.

Agreed.  Main reason for mentioning it was because you could fully containerize the build process and build environment (and ensure build/deploy flavors match exactly).  This would also support identical build process with and without Github Actions as the build tool and eliminate requirement for `jdk` be installed on the host that's doing the build.